packages/storybook/package.json

```diff
@@ -4,33 +4,47 @@
   "private": true,
   "scripts": {
     "dev": "storybook dev -p 6006",
-    "build": "storybook build --docs",
+    "build": "storybook build",
     "preview-storybook": "serve storybook-static",
     "clean": "rm -rf .turbo && rm -rf node_modules",
     "lint": "eslint ./stories/*.stories.tsx"
   },
   "prettier": "@dailydotdev/prettier-config",
   "dependencies": {
     "@dailydotdev/shared": "*",
+    "@tanstack/react-query": "^4.36.1",
+    "extension": "*",
+    "next": "^12.2.2",
+    "next-router-mock": "^0.9.13",
     "react": "18.3.1",
-    "react-dom": "18.3.1"
+    "react-dom": "18.3.1",
+    "webextension-polyfill": "^0.10.0"
   },
   "devDependencies": {
+    "@chromatic-com/storybook": "^1",
     "@dailydotdev/eslint-config": "*",
     "@dailydotdev/eslint-plugin-daily-dev-eslint-rules": "*",
     "@dailydotdev/prettier-config": "*",
-    "@storybook/addon-actions": "^7.6.1",
-    "@storybook/addon-designs": "^7.0.9",
-    "@storybook/addon-docs": "^7.6.10",
-    "@storybook/addon-essentials": "^7.6.1",
-    "@storybook/addon-links": "^7.6.1",
-    "@storybook/addon-themes": "^7.6.1",
-    "@storybook/react": "^7.6.1",
-    "@storybook/react-vite": "^7.6.1",
-    "@storybook/theming": "^7.6.10",
+    "@storybook/addon-actions": "^8.1.11",
+    "@storybook/addon-designs": "^8.0.2",
+    "@storybook/addon-docs": "^8.1.11",
+    "@storybook/addon-essentials": "^8.1.11",
+    "@storybook/addon-interactions": "^8.1.11",
+    "@storybook/addon-links": "^8.1.11",
+    "@storybook/addon-themes": "^8.1.11",
+    "@storybook/react": "^8.1.11",
+    "@storybook/react-vite": "^8.1.11",
+    "@storybook/theming": "^8.1.11",
+    "@svgr/plugin-jsx": "^8.1.0",
+    "@svgr/plugin-svgo": "^8.1.0",
+    "@types/node": "^16.11.41",
```

Comment:
This also doesn't match what we have, not sure it was intended?

Reply:
Not intended, just moving to 20.14. Thanks!